src/gui/editors/SongEditor.cpp

```diff
@@ -101,6 +101,12 @@ SongEditor::SongEditor( Song * song ) :
 			 m_positionLine, SLOT( update() ) );
 	connect( this, SIGNAL( zoomingValueChanged( float ) ),
 			 m_positionLine, SLOT( zoomChange( float ) ) );
+			 
+	// ensure loop markers snap to correct size
+	connect(m_snappingModel, &ComboBoxModel::dataChanged,
+		[=]() { m_timeLine->updateSnapSize(getSnapSize()); });
```

Comment:
Implicit capture of `this` using `[=]` is deprecated in C++20. Not really a problem since we're only on C++17, but explicit capture would be better for the future.

Reply:
Fixed!